GitVersion.yml

```diff
@@ -2,6 +2,4 @@ assembly-versioning-scheme: Major
 next-version: 6.0
 branches:
   develop:
-    tag: alpha
-  releases?[/-]:
```

Comment:
Got it!

@Particular/nservicebus-maintainers so we need to make this change to core as well once we upgrade to GV 4 

https://github.com/Particular/NServiceBus/blob/develop/GitVersion.yml

@SimonCropp do you know if there is a equivalent setting in v4? (assuming it's not the default)